recipes/nn.rb

```diff
@@ -107,6 +107,10 @@
   mode "700"
 end
 
+deps = ""
+if exists_local("ndb", "mysqld") 
+  deps = "mysqld.service"
```

Comment:
Influxdb needs to be up before the namenode is started, otherwise you won't get any metric.

Reply:
Ok, i made the fix.